cmake 3.15 vs 3.10 has sed error for hh_update

Windows msys2 /mingw64/bin/cmake version 3.15.4 emits
```
	sed "'s/ RANGE minf/ GLOBAL minf/'" ...
```
which gives
```
[ 84%] Update hh.mod for CoreNEURON compatibility
sed: -e expression #1, char 1: unknown command: `''
make[2]: *** [CMakeFiles/hh_update.dir/build.make:58: CMakeFiles/hh_update] Error 1
```
whereas Linux cmake version 3.10.2 emits
```
sed 's/\ RANGE\ minf/\ GLOBAL\ minf/'
```
How does one work around this issue?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -311,6 +311,8 @@ else()
   set(GLOBAL_VAR_TOGGLE_COMMAND "'s/ RANGE minf/ GLOBAL minf/'")
   set(TABLE_VAR_TOGGLE_COMMAND "'s/ :TABLE minf/ TABLE minf/'")
 endif()
+separate_arguments(GLOBAL_VAR_TOGGLE_COMMAND UNIX_COMMAND "${GLOBAL_VAR_TOGGLE_COMMAND}")
+separate_arguments(TABLE_VAR_TOGGLE_COMMAND UNIX_COMMAND "${TABLE_VAR_TOGGLE_COMMAND}")
 add_custom_target(
   hh_update
   COMMAND sed ${GLOBAL_VAR_TOGGLE_COMMAND} ${CMAKE_SOURCE_DIR}/src/nrnoc/hh.mod >
@@ -320,7 +322,8 @@ add_custom_target(
   COMMAND ${CMAKE_COMMAND} -E copy_if_different ${CMAKE_BINARY_DIR}/hh.mod.2
           ${CMAKE_SOURCE_DIR}/src/nrnoc/hh.mod
   COMMAND ${CMAKE_COMMAND} -E remove ${CMAKE_BINARY_DIR}/hh.mod.1 ${CMAKE_BINARY_DIR}/hh.mod.2
-  COMMENT "Update hh.mod for CoreNEURON compatibility")
+  COMMENT "Update hh.mod for CoreNEURON compatibility"
+  VERBATIM)
 add_dependencies(nrniv hh_update)
 
 # =============================================================================

